tools/Google.Cloud.Tools.ReleaseManager/CheckVersionCompatibilityCommand.cs

```diff
@@ -50,68 +50,41 @@ public void Execute(string[] args)
             {
                 Console.WriteLine($"Checking compatibility for {api.Id} version {api.Version}");
                 var prefix = api.Id + "-";
-                var previousVersions = tags
+                var lastVersion = tags
                     .Where(tag => tag.StartsWith(prefix))
                     .Select(tag => tag.Split(new char[] { '-' }, 2)[1])
                     .Where(v => !v.StartsWith("0")) // We can reasonably ignore old 0.x versions
                     .Select(StructuredVersion.FromString)
                     .OrderBy(v => v)
-                    .ToList();
+                    .LastOrDefault();
+
+                if (lastVersion is null)
+                {
+                    Console.WriteLine("No previous versions released; ignoring.");
+                }
```

Comment:
Darn - missed a return statement here :(

Reply:
I'll add it in a separate PR